Versions-ios.plist.in

```diff
@@ -101,5 +101,15 @@
 			<string>2.0</string>
 		</dict>
 	</dict>
+	<key>Features</key>
+	<array>
+		<string>mlaunch-launchdevbundleid</string>
+		<string>mlaunch-observe-extension</string>
+		<string>mlaunch-launch-simulator</string>
+		<string>mlaunch-install-progress</string>
+		<string>http-client-handlers</string>
+		<string>mono-symbol-archive</string>
+		<string>sgen-concurrent-gc-experimental</string>
```

Comment:
That's tracking one-too-many issues. @jstedfast can you remove both `<string>sgen-concurrent-gc-experimental</string>` lines from the PR so the other feature can be tested. We'll do another one once for the last item once we have results from QA. Thanks.